<!-- ROS launch file -->
<launch>
    <arg name="node_name"       default="wp_mission" />
    <arg name="launch_prefix"   default="xterm -hold -geometry 105x19 -e" />

    <!-- the node -->
    <node name="$(arg node_name)" pkg="tt_mavros_wp_mission" type="global_pos_mission" launch-prefix="$(arg launch_prefix)">
        <!-- remapping -->
    </node>
</launch>
